packages/ui/src/fields/Relationship/Input.tsx

```diff
@@ -768,7 +768,11 @@ export const RelationshipInput: React.FC<RelationshipInputProps> = (props) => {
               }}
               onMenuOpen={() => {
                 if (appearance === 'drawer') {
-                  openListDrawer()
+                  // TODO: This timeout is only necessary for inline blocks in the lexical editor
+                  // and when the devtools are closed. Temporary solution, we can probably do better.
+                  setTimeout(() => {
+                    openListDrawer()
+                  }, 50)
```

Comment:
I got to 50ms through trial and error. At 40ms, the bug still occurred. The lag is imperceptible to me, so for now, it seems good enough as a temporary solution.

Reply:
>  Interestingly, at least in Chrome, this only happens with DevTools closed.

Ah, I missed this detail